frontend-react/e2e/spec/all/idletimeout.spec.ts

```diff
@@ -37,27 +36,28 @@ const test = baseTest.extend<OrganizationPageFixtures>({
             frontendWarningsLogPath,
             isFrontendWarningsLog,
         });
+        await page.page.clock.install();
```

Comment:
Not sure how time is persisted in these tests, but is it worth cleaning up afterwards like so:

```
test.afterEach(async ({ page }) => {
    await page.clock.uninstall();
});
```